packages/newsroom-signup/src/SmartContract/index.tsx

```diff
@@ -23,18 +22,12 @@ const ContinueButtonContainer = styled.div`
   justify-content: center;
 `;
 
-export class SmartContract extends React.Component<SmartContractProps, SmartContractState> {
-  constructor(props: SmartContractProps) {
-    super(props);
-    this.state = {
-      currentStep: 0,
-    };
-  }
+export class SmartContract extends React.Component<SmartContractProps> {
   public renderButtons(): JSX.Element | void {
-    if (this.state.currentStep === 0) {
+    if (this.props.currentStep === 0) {
       return (
         <ContinueButtonContainer>
-          <ContinueButton onClick={this.goNext} size={buttonSizes.MEDIUM_WIDE}>
+          <ContinueButton onClick={() => this.props.navigate(+1)} size={buttonSizes.MEDIUM_WIDE}>
```

Comment:
"+1" as an arg? shouldnt it just be 1?

Reply:
The + doesn't do anything (I mean it just turns `1` into a number), just kind of sounds nice `navigate(+1)` vs `navigate(-1)`, but yeah it's unnecessary, will remove.